src/signals/incident/services/custom-validators/index.js

```diff
@@ -44,3 +50,8 @@ export const validatePhoneNumber = control => {
 
   return { custom: 'Ongeldig telefoonnummer, alleen cijfers, spaties, haakjes, + en - zijn toegestaan.' };
 };
+
+export const validateRequired = message =>
```

Comment:
This is a factory not a validator, right? I would use `create` prefix or Factory suffix here.